security/urls.py

```diff
@@ -1,9 +1,15 @@
-from django.conf.urls import patterns, url
+import django
+from django.conf.urls import url
 
 from security import views
 
-
-urlpatterns = patterns(
-    '',
-    url('^/csp-report/$', views.csp_report),
-)
+if django.VERSION >= (1, 10):
+    urlpatterns = [
+        url('^/csp-report/$', views.csp_report),
+    ]
+else:
+    from django.conf.urls import patterns
```

Comment:
Patterns was actually deprecated as of 1.8, but I don't know if anyone will care

https://docs.djangoproject.com/en/1.8/ref/urls/#patterns